bctw-api/src/critterbase/critterbase_api.ts

```diff
@@ -0,0 +1,31 @@
+import axios, { AxiosError, AxiosResponse, Method } from "axios";
+import { formatAxiosError } from "../utils/error";
+
+
+const critterBaseRequest = async (method: Method, url: string, body: any = {}) => {
+    let errStr = '';
+    const header = {headers: { 'API-KEY' : process.env.CRITTERBASE_API_KEY }}
+    if(!process.env.CRITTERBASE_URL) {
+        throw Error("Can't make requests unless CRITTERBASE_URL is set.")
+    }
+    switch (method.toUpperCase()) {
+        case "GET": 
+            return await axios
+            .get(process.env.CRITTERBASE_URL + url, header)
+            .catch((err: AxiosError) => {
+                errStr = formatAxiosError(err);
```

Comment:
If you wrap the axios request in the query function the query will handle the axios error now